outlines/generate/api.py

```diff
@@ -529,3 +567,22 @@ def stream(
             self.sampling_params,
             **model_specific_params,
         )
+
+
+def apply_chat_template_util(model, prompts: Union[str, List[str]]) -> List[str]:
+    from outlines.models.transformers import TransformerTokenizer
+
+    if isinstance(prompts, str):
+        prompts = [prompts]
```

Comment:
I think the signature should be `List[str]` -> `List[str]` and raise an error if a list isn't passed. In `transformers.py`, this function is called after the prompts are normalized to 2D anyways.